agentlocal/agentlocal.go

```diff
@@ -32,9 +32,9 @@ import (
 )
 
 // SetTransport configures transport for accessing local pmm-agent API.
-func SetTransport(ctx context.Context, debug bool) {
+func SetTransport(ctx context.Context, debug bool, pmmAgentListenPort uint32) {
```

Comment:
If we let pmmAgentListenPort its more clear from where value comes from since we have another flags with port in name. What do you think is better? I am OK wuth both solutions.

Reply:
Changed.